wallet/wallet.go

```diff
@@ -3224,6 +3224,18 @@ func (w *Wallet) PublishTransaction(tx *wire.MsgTx) error {
 	_, err = server.SendRawTransaction(tx, false)
 	switch {
 	case err == nil:
+		switch w.chainClient.(type) {
+		// For neutrino we need to trigger adding relevant tx manually
+		// because for spv client - tx data isn't received from sync peer.
+		case *chain.NeutrinoClient:
+			rec, err := wtxmgr.NewTxRecordFromMsgTx(tx, time.Now())
```

Comment:
Could just use the same `txRec` found above before `SendRawTransaction`.

Reply:
you are right, didn't notice it after move from another method, chnaged